.github/workflows/update-resources.yaml

```
@@ -0,0 +1,66 @@
+name: Updates SDK resources based on api-definitions
+
+on: 
+  push:
+  workflow_dispatch:
+    inputs:
+      api-definitions-branch:
+        description: 'api-definitions branch triggering the current workflow'
+        required: true
+        default: 'master'
+      auto-pr-title:
+        description: 'title of the PR to be created'
+        required: true
+        default: 'Auto generated PR'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout api-definitions repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'Rebilly/api-definitions'
+          path: 'api-definitions'
+          # Here we need to use the name of the branch that triggered this workflow (instead of master)
+          ref: '${{ github.event.inputs.api-definitions-branch }}'
+      - uses: actions/setup-node@v2
+        with:
+          # We need node LTS to run rebilly-SDK-generator
+          node-version: '14'
+      - name: Builds openAPI json schemas
+        working-directory: 'api-definitions'
+        run: |
+          npm install
+          npx openapi bundle -o dist --ext json
+      - name: Checkout rebilly-js-sdk repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          path: 'rebilly-js-sdk'
+      - name: Checkout sdk generator repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'Rebilly/rebilly-sdk-generator'
+          path: 'rebilly-sdk-generator'
+          # Here we will use master once we have merged
+          ref: 'master'
```

```suggestion
          ref: 'main'
```
To prepare for name change